_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -20,7 +20,7 @@ class AnnualIncomeSection extends React.Component {
 
     if (this.props.external.receivesVaPension === true) {
       notRequiredMessage = (
-        <p>
+        <p id="notRequiredMessage">
```

Comment:
I think it's better not to use an id, and instead a class if you need it for testing purposes. The test should still find as many instances as there are in that section because it is only rendering that section, so no need to use an id.